README.md

```diff
@@ -5,63 +5,62 @@
 
 A simple, lightweight, no-dependencies JavaScript image lightbox.
 
-* [Installation](#installation)
-* [Usage](#usage)
-	* [LuminousGallery Usage](#luminousgallery-usage)
-* [Options / Defaults](#options-defaults)
-	* [LuminousGallery Options / Defaults](#luminousgallery-options-defaults)
-* [Browser Support](#browser-support)
-* [Theming](#theming)
-* [Meta](#meta)
-
+- [Installation](#installation)
+- [Usage](#usage) \* [LuminousGallery Usage](#luminousgallery-usage)
```

Comment:
Hey @frederickfogerty I just noticed that Prettier seems to have messed up our nested bullets here.

Reply:
After investigation, it seems to be a bug in the parser that prettier uses for markdown parsing. See more [here](https://github.com/prettier/prettier/issues/3223). I've fixed this by using spaces rather than tabs for this list. Will submit a PR now.